apps/meteor/server/models/raw/Roles.ts

```diff
@@ -157,6 +157,15 @@ export class RolesRaw extends BaseRaw<IRole> implements IRolesModel {
 		return this.find(query, options || {});
 	}
 
+	findCustomUserRoles(options?: FindOptions<IRole> | undefined): FindCursor<IRole> {
+		const query = {
+			scope: 'Users' as IRole['scope'],
```

Comment:
```suggestion
		const query: Filter<IRole> = {
			scope: 'Users',
```

Reply:
It's better to declare the type of the variable here and let ts check if the value makes sense, instead of casting. Casting means overriding a type (not changing) (in other words "shut up typescript, I know this type better than you") - which should be a last resort IMO. More so if you look at the scope field, it already has `Users` - you just need to help the typechecker a little bit in this case to understand the exact type name.

Reply:
Check [this](https://www.typescriptlang.org/play?#code/C4TwDgpgBAYg9nKBeKBvAUFLUCWATALigGdgAnHAOwHMBuTbAQ2oiMoFcBbAIwjKgA+JclTroAvvXQBjOJVJQAZgmRoGWfEQDkxOJwj4tAGnVRmrKACYAjBKkB6eyQjQAFnADuUYK+h4IwBDSgXje4G6MxFAABubRZlEAVuwKjFAcPHzojlAQAG4QlN7u7NSuUB7QnIwgUK6MBVAARFSBlP54TbjA3oi8sAgA2lrmWgC66KCQUAAqyogoUxBwikoItFDZTthQAHoA-OhbdXzQlcKMZD3ctYEANnei3lEe9T0A1lShK1B4jMBpACSFUYlGAx3mll6wjgZAgxwAFJ92lAfixKHxGI8AF5PHzQJb7ACUMjkCkhRHgCzUO00UB0egMeGMpnMRBsdiOsnkPTgrko1IwtMI9N0+kMJh2bKUWOI0EiUHYlHeAo8RQVVOGozGUBy7i80kiwCedwCURSv0QPlEUR8-wqfjkWh6GOkEGIxEuODutQ8oPB4iOS1mfIFqiWP1DcHoOR2ByD4VmkPD4R+kJj2yw8aAA)